sdk/src/main/java/com/bugsnag/android/Session.java

```diff
@@ -55,12 +63,14 @@ int getHandledCount() {
         return handledCount.intValue();
     }
 
-    void incrementHandledErrCount() {
+    Session incrementHandledErrCount() {
```

Comment:
I don't think it's obvious that these methods increment the current session and also return a copy of the incremented session.

Perhaps `copySession` should just be called independently

Reply:
I've renamed this `incrementHandledAndCopy`, and also added some JavaDoc.